core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1939,4 +1939,8 @@ TimeoutException coordinatorTimeoutWaitingForView(int expectedViewId, int curren
 
    @Message(value = "RELAY2 not found in ProtocolStack. Unable to do cross-site requests.", id = 571)
    CacheConfigurationException crossSiteUnavailable();
+
+   @LogMessage(level = WARN)
+   @Message(value = "IndexMode is deprecated and is automatically detected when indexing is enabled. Please replace this attribute with \"enabled='true'\"", id = 572)
```

Comment:
This needs to be reworded slightly because we don't really have an ```IndexMode```, just an ```Index```...

Reply:
Right, IndexMode is the "logical" name, I will rephrase it everywhere